### PR TITLE
[MRG+1] FIX: bug non passed parameter

### PR DESCRIPTION
**Pull request by kingjr**
Pretty bad mistake on my part. Here's a fix

### CODE REVIEW

**Comment by dengemann** [comment]
Impeccable.

File: mne/decoding/tests/test_time_gen.py [hunk 1]
```diff
@@ -333,5 +335,11 @@ def test_decoding_time():
     assert_equal("<TimeDecoding | fitted, start : -0.200 (s), stop : 0.499 "
                  "(s), predicted 14 epochs,\n scored (accuracy_score)>",
                  '%s' % tg)
+    # Test with regressor
+    clf = SVR()
+    cv = KFold(len(epochs))
+    y = np.random.rand(len(epochs))
+    tg = TimeDecoding(clf=clf, cv=cv)
```
Comment (agramfort) [inline mne/decoding/tests/test_time_gen.py]: the code should detect automatically that clf is a regressor and switch to KFold. See how it's done in sklearn.

the code should not break when cv is not set and clf is a regressor.

Comment (dengemann) [inline mne/decoding/tests/test_time_gen.py]: The other important problem here was that a parameter was not passed. It seems the auto-detection is a separate issue.

Comment (agramfort) [inline mne/decoding/tests/test_time_gen.py]: yes but for me it's also a bug.

Comment (dengemann) [inline mne/decoding/tests/test_time_gen.py]: yes it is.

Comment (kingjr) [inline mne/decoding/tests/test_time_gen.py]: Yes, it's already noted in https://github.com/mne-tools/mne-python/issues/2820. Let's deal with issues separately?

Comment (agramfort) [inline mne/decoding/tests/test_time_gen.py]: ok. Was this bug part of the release? if so add an entry in what's new please

thx

File: doc/whats_new.rst [hunk 2]
```diff
@@ -37,6 +37,8 @@ BUG
 
     - Time cropping functions no longer use relative tolerances when determining the boundaries by `Jaakko Leppakangas`_
 
+    - Fix :func:`mne.decoding.TimeDecoding` to allow specifying `clf` by `Jean-Remi King`_
```
Comment (Eric89GXL) [inline doc/whats_new.rst]: the single-backticks on `clf` here won't be made into a proper reference, so better to use double-backticks so it gets rendered as code

**Comment by Eric89GXL** [comment]
Thanks @kingjr

**Pull request merged**